src/gamestate/system_state.cpp

```diff
@@ -2621,8 +2621,16 @@ void state::load_scenario_data() {
 
 	military::recover_org(*this);
 
-	if(err.accumulated_errors.length() > 0)
-		window::emit_error_message(err.accumulated_errors, err.fatal);
+	if(!err.accumulated_errors.empty()) {
+		if(err.fatal) {
```

Comment:
I really don't want this. In just two days I will be writing the launcher, and I will want it to be able to report errors in its own way